Mage.Sets/src/mage/cards/b/BeneathTheSands.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
There's a gen-card.pl file in mage/utils that you can use so all cards have a similar format.  Also, if using netbeans, you can do a Source -> Format then Source -> Fix Imports.

Looks ok other than that!